core/client/fs/src/main/java/alluxio/client/MetaMasterClient.java

```diff
@@ -11,17 +11,26 @@
 
 package alluxio.client;
 
+import alluxio.wire.ConfigInfo;
 import alluxio.wire.MasterInfo;
 import alluxio.wire.MasterInfo.MasterInfoField;
 
 import java.io.Closeable;
 import java.io.IOException;
+import java.util.List;
 import java.util.Set;
 
 /**
  * Interface for a meta master client.
  */
 public interface MetaMasterClient extends Closeable {
+  /**
+   * Gets the runtime configuration information.
+   *
+   * @return a list of configuration information
+   */
+  List<ConfigInfo> getConfigInfoList() throws IOException;
```

Comment:
We can remove "List" from the name - the type already shows that it's a list
"Info" also doesn't really add anything - what's the difference between `getConfig` and `getConfigInfo`?

Reply:
config is a type of info, but blocks/masters/workers are objects or servers, not types of info. I think it's fine to shorten to just `getConfig` if you prefer it

Reply:
Changed to GetConfiguration because It is consistent with what AlluxioMasterInfo, AlluxioProxyInfo and AlluxioWorkerinfo used!
They all have public Map<String, String> getConfiguration() {return mConfiguration;}
```
struct ConfigProperty {
  1: string name
  2: string source
  3: string value
}

struct GetConfigurationTOptions{}
struct GetConfigurationTResponse{
  1: list<ConfigProperty> configList
}

service MetaMasterClientService extends common.AlluxioService {
  /**
   * Returns a list of Alluxio runtime configuration information.
   */
  GetConfigurationTResponse getConfiguration(
    /** the method options */ 1: GetConfigurationTOptions options,
  ) throws (1: exception.AlluxioTException e)
}
```